LibVlcSharp.WPF.Sample/Controls.xaml.cs

```diff
@@ -0,0 +1,55 @@
+﻿using LibVLCSharp.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+
+namespace LibVLCSharp.WPF.Sample
+{
+    public partial class Controls : UserControl
+    {
+        Example1 parent;
+
+        public Controls(Example1 Parent)
+        {
+            parent = Parent;
+
+            InitializeComponent();
+            
+            PlayButton.Click += PlayButton_Click;
+            StopButton.Click += StopButton_Click;
+        }
+
+        private void StopButton_Click(object sender, RoutedEventArgs e)
+        {
+            parent.Dispatcher.Invoke(() => {
```

Comment:
off topic/review question: Do you think it'd make sense for the binding to offer a dispatcher callback registration, so users dont have to do just that on most calls?

Reply:
I'm not sure how to implement it inside the Control to work for all calls.

Reply:
seems like it's not needed anymore... just tested it without invoke (ThreadIDs are the same)

Reply:
@mfkl : I think it might be interesting to avoid a lot of questions like this one : https://github.com/ZeBobo5/Vlc.DotNet/issues/444

Reply:
Yes, it's not needed : You are in a UI event handler, which runs on the dispatcher.